build.cmd

```diff
@@ -152,6 +152,14 @@ set Platform=
 if NOT "%__DotNetCliPath%" == "" goto SetupManagedBuild
 
 set "__DotNetCliPath=%__RootBinDir%\tools\cli"
+if "%__CleanBuild%"=="1" (
```

Comment:
We already clean the bin/Product/<OS><arch><flavor> directory for bin, Intermediate and Logs.